<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Record</class>
 <widget class="QWidget" name="Record">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(3, 127, 119);
}
QGroupBox{
font: 14pt &quot;楷体&quot;;
border: 1px solid white;
border-radius:8px;
margin-top:10px;
}
QGroupBox:title{
color:white;
subcontrol-origin: margin;
left: 5px;
}
QToolButton{
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
border:1px solid;
border-radius:10px;
border-color: rgb(0, 0, 0);
background-color: rgb(43, 79, 102);
}
QToolButton:pressed{
font: 16pt &quot;楷体&quot;;
background-color: rgb(43, 79, 102);
}

QPushButton{
width:100px;
height:40px;
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
border:1px solid;
border-radius:10px;
border-color: rgb(0, 0, 0);
background-color: rgb(43, 79, 102);
}
QPushButton:hover{
background-color: rgb(0, 170, 255);
}
QLabel{
font: 16pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
}
QComboBox{
font: 16pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
border:1px solid white;
background-color: rgb(3, 127, 119);
}
QComboBox QAbstractItemView::item{
font: 16pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
}

QDateTimeEdit{
font: 16pt &quot;楷体&quot;;
border:1px solid white;
color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>-5</y>
     <width>1120</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="cbBoxHost">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>20</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>网络地址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>20</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>从</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>到</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cbBoxType">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>20</y>
      <width>116</width>
      <height>40</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>全部故障</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>过流故障</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>缺相故障</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>过压故障</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>欠压故障</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>通讯故障</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>供电中断</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>56</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>故障：</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dTEditStart">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy年MM月dd日 HH:mm</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dTEditStop">
    <property name="geometry">
     <rect>
      <x>855</x>
      <y>20</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="displayFormat">
     <string>yyyy年MM月dd日 HH:mm</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>55</y>
     <width>1120</width>
     <height>600</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>15</y>
      <width>1110</width>
      <height>536</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
font: 13pt &quot;楷体&quot; rgb(255, 255, 255);
border:1px  none white;
gridline-color: rgb(0, 0, 0);
background-color: rgb(3, 127, 119);
}</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnDelete">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>555</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
    <property name="icon">
     <iconset resource="../MainPage/image.qrc">
      <normaloff>:/Image/recycle.png</normaloff>:/Image/recycle.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnQuery">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>555</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
    <property name="icon">
     <iconset resource="../MainPage/image.qrc">
      <normaloff>:/Image/search.png</normaloff>:/Image/search.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../MainPage/image.qrc"/>
 </resources>
 <connections/>
</ui>
